ext/zip/config.m4

```diff
@@ -1,91 +1,47 @@
 dnl config.m4 for extension zip
 
 PHP_ARG_ENABLE([zip],
-  [for zip archive read/writesupport],
+  [for zip archive read/write support],
   [AS_HELP_STRING([--enable-zip],
     [Include Zip read/write support])])
 
-PHP_ARG_WITH([libzip],
```

Comment:
With `--with-libzip` gone, we should probably rename `--enable-zip` to `--with-zip`.

Reply:
Okay.